onebusaway-android/src/main/java/org/onebusaway/android/directions/util/TripRequestBuilder.java

```diff
@@ -196,7 +199,7 @@ public TripRequestBuilder setModeSetById(int id) {
     public int getModeSetId() {
         // IF bike mode is selected in the trip plan additional preferences but bikeshare is not enabled use the default mode (TRANSTI)
         if (BIKESHARE_MODE == mModeId && !Application.isBikeshareEnabled()) {
-            return TRANSIT_MODE;
+            return TRANSIT_AND_BIKE_MODE;
```

Comment:
@carvalhorr It seems here either the comment is wrong or this should be `TRANSIT_MODE`.  Could you please resolve the two so they match?